MVCapplication/src/main/webapp/WEB-INF/resources/js/payment.js

```diff
@@ -0,0 +1,73 @@
+
+var $form = $('#payment-form');
+$form.find('.subscribe').on('click', payClicked);
+
+function payClicked(e) {
+    e.preventDefault();
+    /* Visual feedback */
+    $form.find('.subscribe').html(validating + ' <i class="fa fa-spinner fa-pulse"></i>').prop('disabled', true);
+    setTimeout(function(){$form.find('.subscribe').html(paymentSuccessful + ' <i class="fa fa-check"></i>'); window.location.href='/profile/getPremium';}, 500);
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') "paymentSuccessful" does not exist. Change its name or declare it so that its usage doesn't result in a "ReferenceError". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS3827)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)